frontend/integration-tests/tests/storage.scenario.ts

```diff
@@ -0,0 +1,62 @@
+import { $$, browser, ExpectedConditions as until, by, element } from 'protractor';
+import { testName, checkLogs, checkErrors } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+import { execSync } from 'child_process';
+
+describe('Add storage is applicable for all workloads', () => {
+
+  const k8sWorkloads = ['replicationcontrollers', 'daemonsets', 'deployments', 'replicasets', 'statefulsets'];
+  const openshiftWorkloads = ['deploymentconfigs'];
+  const resourceObjs = browser.params.openshift === 'true' ? openshiftWorkloads.concat(k8sWorkloads) : k8sWorkloads;
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  afterAll(() => {
+    resourceObjs.forEach((resourcetype) => {
+      try {
+        execSync(`kubectl delete --cascade ${resourcetype} example -n ${testName}`);
+      } catch (error) {
+        console.error(`Failed to delete ${resourcetype} example:\n${error}`);
+      }
+    });
+  });
+
+  resourceObjs.forEach((resourcetype) => {
+    describe(resourcetype, () => {
+      const pvcName = `${resourcetype}-pvc`;
+      const pvcSize = '1';
+      const mountPath = '/data';
+      it(`create a ${resourcetype} resource`, async() => {
+        await crudView.createNamespacedResourceWithDefaultYaml(resourcetype);
+        expect(crudView.errorMessage.isPresent()).toBe(false);
+      });
+      it(`add storage to ${resourcetype}`, async() => {
+        if (resourcetype === 'deploymentconfigs' || resourcetype === 'deployments') {
```

Comment:
Why only dc and deployments? Why not all workloads?

Reply:
During the test, I easily met conflict error only when add storage to dc/deployment, so I wait until the previous deployment finished. Seems other workloads are stable when yaml is updated.